jobs/config_test.py

```diff
@@ -567,6 +567,8 @@ def test_valid_job_config_json(self):
 
                     if shared_builds or node_e2e:
                         expected = 0
+                    elif 'ingress' in job:
```

Comment:
I'm hoping we will not have another ingress job that actually requires two `--extract` :)